101-vpn-custom-ipsec-policy/azuredeploy.json

```diff
@@ -0,0 +1,203 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location" : {
+            "defaultValue": "[resourceGroup().location]",
+            "type": "string"
+        },
+        "vpnGateway_Name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of existing Virtual Network Gateway to deploy the connection to"
+            }
+        },
+        "localGateway_Name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of existing Local Network Gateway to deploy the connection to"
+            }
+        },
+        "vpnName": {
+            "type": "string",
+            "metadata": {
```

Comment:
move all the metadata properties to the bottom of the param declaration

Reply:
addressed in commit 604d1ab